src/main/java/com/my/FoodTruckBackend/HelloWorldController.java

```diff
@@ -0,0 +1,13 @@
+package com.my.FoodTruckBackend;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HelloWorldController {
+
+    @GetMapping("/helloWorld")
+    public String helloWorld() {
+        return "HELLO WORLD!!!";
+    }
+}
```

Comment:
I don't think this API is being used. Is it needed ?

Reply:
This was only for testing and has been removed